### PR TITLE
[ACS-7427] Process Services improvements and cleanup

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

ACS-7427

**What is the new behaviour?**

- ACS-7427: Break Process Comments dependency on Material Module
- ACS-7426: Break People dependency on Material Module
- Rework and cleanup  `App List Component`
- Rework and cleanup `People Component`
- migrate process comments to standalone
- break MaterialModule dependency for process comments
- lots of typing fixes
- use native JS-API interfaces in process components (cleanup custom models/wrappers)
- dead code cleanup
- update documentation

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

[ACS-7427]: https://hyland.atlassian.net/browse/ACS-7427?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ACS-7426]: https://hyland.atlassian.net/browse/ACS-7426?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/process-services/src/lib/process-comments/mock/comment-process-service.mock.ts [hunk 1]
```diff
@@ -18,34 +18,41 @@
 import { Injectable } from '@angular/core';
 import { Observable, from, of } from 'rxjs';
 import { map } from 'rxjs/operators';
-import { CommentModel, UserProcessModel, CommentsService } from '@alfresco/adf-core';
-import { fakeUser1 } from '../mock/comment-process.mock';
+import { CommentModel, CommentsService } from '@alfresco/adf-core';
+import { UserProcessModel } from '@alfresco/adf-process-services';
+
+export const fakeUser1 = { id: 1, email: 'fake-email@dom.com', firstName: 'firstName', lastName: 'lastName' };
```
Comment (nikita-web-ua) [inline lib/process-services/src/lib/process-comments/mock/comment-process-service.mock.ts:24]: missing `avatarId` which defined as compulsory

Comment (DenysVuika) [reply lib/process-services/src/lib/process-comments/mock/comment-process-service.mock.ts:24]: thanks, good catch, I will fix the missing typings in the future

**Review by nikita-web-ua — APPROVED** [review]

**Review by DaryaBalvanovich — APPROVED** [review]

**Review by DaryaBalvanovich — APPROVED** [review]

File: docs/core/services/people-process.service.md [hunk 2]
```diff
@@ -2,69 +2,48 @@
...
-[Task Actions API](https://github.com/Alfresco/alfresco-js-api/blob/master/src/alfresco-activiti-rest-api/docs/TaskActionsApi.md#involveUser)
-(for `involveUserWithTask` and `removeInvolvedUser`), the
-[User Workflow API](https://github.com/Alfresco/alfresco-js-api/blob/master/src/alfresco-activiti-rest-api/docs/UsersWorkflowApi.md#getUsers)
-(for `getWorkflowUsers`) and the
-[User API](https://github.com/Alfresco/alfresco-js-api/blob/master/src/alfresco-activiti-rest-api/docs/UserApi.md#getuserprofilepictureurl)(for `getUserImage`).
+    -   **Returns** `Observable<LightUserRepresentation>` - Empty response when the update completes
 
 ## See also
 
 -   [User process model](../models/user-process.model.md)
```
Comment (nikita-web-ua) [inline docs/core/services/people-process.service.md:48]: should be deleted, and `user-process.model.md` itself too?

File: lib/process-services/src/lib/app-list/apps-list.component.ts [hunk 3]
```diff
@@ -30,7 +29,7 @@ import { MatCardModule } from '@angular/material/card';
 import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
 import { TranslateModule } from '@ngx-translate/core';
 
-const DEFAULT_TASKS_APP: string = 'tasks';
+// const DEFAULT_TASKS_APP: string = 'tasks';
```
Comment (nikita-web-ua) [inline lib/process-services/src/lib/app-list/apps-list.component.ts:32]: intended comment?

**Pull request merged**